test/extended/deployments/deployments.go

```diff
@@ -171,12 +189,10 @@ var _ = g.Describe("deploymentconfigs", func() {
 				secondDeploymentName := deployutil.DeploymentNameForConfigVersion(name, 2)
 				secondDeployerExists := len(deploymentNamesToDeployers[secondDeploymentName]) == 1
 
-				return hasCorrectLatestVersion &&
-					secondDeploymentExists &&
-					firstDeployerRemoved && secondDeployerExists, nil
+				return secondDeploymentExists
 			}
-
 			err = wait.PollImmediate(500*time.Millisecond, 10*time.Second, condition)
+			printDebugInfo(err, oc, name)
```

Comment:
I cannot see the debug info from the failure trap on the test above working, probably because the test is not returning but the test framework exits without running the defer? We probably need an `AfterEach` handler.